### PR TITLE
profile page - needs refactoring

### PR DESCRIPTION
**Pull request by sophiabrent**

### CODE REVIEW

**Comment by zineanteoh** [comment]
looks ready to merge from my end -- @sophiabrent @ethanratnofsky please review

File: src/screens/profile/ProfileScreen.styles.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  name: {
+    fontFamily: Fonts.COMFORTAA_REGULAR,
+    fontSize: 24,
+    color: COLORS.WHITE,
+    marginTop: 5,
+  },
+  number: {
+    fontFamily: Fonts.COMFORTAA_REGULAR,
+    fontSize: 14,
+    color: '#A6A6A6',
```
Comment (ethanratnofsky) [inline src/screens/profile/ProfileScreen.styles.ts:68]:
```suggestion
    color: COLORS.GRAY,
```

File: src/screens/profile/ProfileScreen.styles.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    backgroundColor: COLORS.NIGHTLIGHT_GRAY,
+    alignItems: 'center',
+    overflowY: 'scroll',
+  },
+
+  // background image
+  backgroundImageContainer: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
```
Comment (ethanratnofsky) [inline src/screens/profile/ProfileScreen.styles.ts:18]: use position absolute sparingly!! probably don't need this here

File: src/screens/profile/ProfileScreen.styles.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  },
+
+  // profile info
+  profileInfoContainer: {
+    position: 'relative',
+    marginTop: height * 0.19,
+    width: width,
+    display: 'flex',
+    alignItems: 'center',
+  },
```
Comment (ethanratnofsky) [inline src/screens/profile/ProfileScreen.styles.ts:50]: this can be positioned absolutely

File: src/screens/profile/ProfileScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                <Text style={ProfileScreenStyles.monthText}>{month}</Text>
+              </View>
+            ))}
+          </View>
+        </View>
+
+        {/* Emergency Contacts button */}
+        <View style={ProfileScreenStyles.emergencyView}>
+          <View style={ProfileScreenStyles.button}>
+            <Button color={'white'} title='View Emergency Contacts' />
```
Comment (ethanratnofsky) [inline src/screens/profile/ProfileScreen.tsx:106]: use Pressable instead of Button

**Review by ethanratnofsky — CHANGES_REQUESTED** [review]

**Comment by zineanteoh** [comment]
image

**Review by ethanratnofsky — APPROVED** [review]

**Pull request merged**